modules/monitoring-granting-user-permissions-using-the-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+[id="granting-user-permissions-using-the-web-console_{context}"]
+= Granting user permissions by using the web console
+
+You can grant users permissions to monitor their own projects, by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* The user account that you are assigning the role to already exists.
+
+.Procedure
+
+. In Administrator perspective in the {product-title} web console, navigate to *User Management* -> *Role Bindings* -> *Create Binding*.
+
+. In the *Binding Type* section, select the "Namespace Role Binding" type.
+
+. In the *Name* field, enter a name for the role binding.
+
+. In the *Namespace* field, select the project where you want to grant the access. For example, select `ns1`.
++
+[IMPORTANT]
+====
+The monitoring role will be bound to the project that you apply in the *Namespace* field. The permissions that you grant to a user by using this procedure will apply only to the selected project.
+====
+
+. Select `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit` in the *Role Name* drop-down menu.
+
+. In the *Subject* section, select *User*.
+
+. In the *Subject Name* field, enter the name of the user. For example, enter `johnsmith`.
+
+. Select *Create* to apply the role binding. If this example, the user `johnsmith` has been assigned permission to set up metrics collection and create alerting rules in the `ns1` project.
```

Comment:
I think
```If this example, the user `johnsmith` has been assigned permission to set up metrics collection and create alerting rules in the `ns1` project.``` could be removed, since `monitoring-rules-view` can create alerting rules

Reply:
@juzhao thank you for highlighting this. I have subsequently removed all of the examples in this procedure because they are superfluous.